engine/xml-config/src/main/resources/configuration.xsd

```diff
@@ -140,6 +171,27 @@
 		</sequence>
 	</complexType>
 
+    <complexType name="remoteComponentsType">
+        <sequence>
+            <element name="server" type="ab:serverType"/>
+        </sequence>
+    </complexType>
+
+    <complexType name="serverType">
+        <sequence>
+            <element name="url" type="string" minOccurs="0" maxOccurs="1">
+                <annotation>
+                    <documentation>
+                        Defines URL to the Datacleaner monitor server
```

Comment:
I believe that the services sub-paths cannot be customized by user on the server-side. If true, so is it fixed, than I would require just the base URL in the conf.xml, the rest should be added by the program.